Berksfile

```diff
@@ -1,11 +1,10 @@
-source 'https://supermarket.getchef.com'
+source 'https://supermarket.chef.io'
 
-cookbook 'certificate'
 cookbook 'firewall', git: 'git@github.com:osuosl-cookbooks/firewall'
-cookbook 'monitoring', git: 'git@github.com:osuosl-cookbooks/monitoring'
 cookbook 'nagios', git: 'git@github.com:osuosl-cookbooks/nagios'
-cookbook 'nginx'
+cookbook 'osl-apache', git: 'git@github.com:osuosl-cookbooks/osl-apache'
```

Comment:
osl-nagios depends on osl-apache even though we're only using the client cookbook (its still in metadata). This will be fixed when we start using the newer upstream nagios/nrpe cookbooks when they're split out.